python/cugraph/graph/test_graph.py

```diff
@@ -75,6 +76,40 @@ def compare_offsets(offset0, offset1):
     return True
 
 
+def compare_graphs(nx_graph, cu_graph):
```

Comment:
compare_graphs() is not exactly correct. You can have 2 identical graphs, one with vertex indices permuted wrt the other and they may have different row_offsets and column_indices. This is the problem of graph isomorphism and does _not_ currently have (such) a trivial solution (it's active research). Perhaps this function should be renamed and comments added to reflect that is a very simplistic heuristics for graph comparison.

Reply:
To clarify... this compare_graphs will return the correct value for it's intended purpose.  If you take the same set of edges, with the same numbering of vertices and create a graph, this function will return true if the resulting graphs match and false if they don't.

The issue @aschaffer is mentioning is a broader graph comparison issue.

Reply:
Thank you very much for the comments, and yes, this is not to test two graphs are isomorphic but to test two graphs are automorphic (isomorphic + vertex mapping between two graphs is an identity mapping). I will add comments to make this point clear.